secio/src/codec/secure_stream.rs

```diff
@@ -209,24 +194,31 @@ where
         cx: &mut Context<'_>,
         buf: &[u8],
     ) -> Poll<io::Result<usize>> {
-        poll_future(cx, self.write_socket(buf))
+        debug!("start sending plain data: {:?}", buf);
```

Comment:
I think it prints too much even for debug, because buf can be huge in application. Please remove it or only print first N bytes of buf.

Reply:
It is the old code. I'm OK to leave it here. But I recommend to limit the visible part of the buf in all debug logs.